mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule Livebook.MixProject do
     ]
   end
 
-  defp extra_applications(:app), do: [:wx]
+  defp extra_applications(:app), do: [:wx, :inets]
```

Comment:
inets is only used at build time. In any case, it is also listed above. :)

```suggestion
  defp extra_applications(:app), do: [:wx]
```

Reply:
I added to remove the warnings during the build.